brage-import/src/main/java/no/sikt/nva/brage/migration/model/content/ResourceContent.java

```diff
@@ -0,0 +1,77 @@
+package no.sikt.nva.brage.migration.model.content;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import java.util.Objects;
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
+public class ResourceContent {
+
+    private List<ContentFile> contentFiles;
+
+    public ResourceContent() {
+
+    }
+
+    public ResourceContent(List<ContentFile> contentFiles) {
+        this.contentFiles = contentFiles;
+    }
+
+    @JsonProperty("contentFiles")
+    public List<ContentFile> getContentFiles() {
+        return contentFiles;
+    }
+
+    public void setContentFiles(List<ContentFile> contentFiles) {
+        this.contentFiles = contentFiles;
+    }
+
+    public ContentFile getContentFileByFilename(String filename) {
```

Comment:
Are you at this point expecting a single match explicitly? Generally, in these cases I prefer nva-commons SingletonCollector.collectOrElse()

Reply:
In short, we are expecting 1 or 0 files. 0 is just as normal situation as 1. We don't want any exception.

Reply:
SingletonCollector.collectOrElse(null)